script/artifact/sonatype-download.sh

```diff
@@ -9,34 +9,29 @@ echo "NEXUS_OBJECT_VERSION=$NEXUS_OBJECT_VERSION"
 echo "NEXUS_OBJECT_CLASSIFIER=$NEXUS_OBJECT_CLASSIFIER"
 
 if [ -z "$NEXUS_OBJECT_CLASSIFIER" ]; then
-    # alapbol a jeloletlen "classifier" nelkulieket keressuk
+    # we are looking for empty classifiers by default
     PATH_PARAM_CLASSIFIER=""
     QUERY_PARAM_CLASSIFIER=""
 else
     PATH_PARAM_CLASSIFIER="-$NEXUS_OBJECT_CLASSIFIER"
     QUERY_PARAM_CLASSIFIER="&c=$NEXUS_OBJECT_CLASSIFIER"
 fi
 
-
 NEXUS_OBJECT_EXTENSION=${NEXUS_OBJECT_EXTENSION:-jar}
 echo "NEXUS_OBJECT_EXTENSION=$NEXUS_OBJECT_EXTENSION"
 NEXUS_DOWNLOAD_OUTPUT_FILE_NAME=${NEXUS_DOWNLOAD_OUTPUT_FILE_NAME:-"$NEXUS_OBJECT_ARTIFACT_ID-$NEXUS_OBJECT_VERSION$PATH_PARAM_CLASSIFIER.$NEXUS_OBJECT_EXTENSION"}
 NEXUS_DOWNLOAD_OUTPUT_FILE_NAME_SHA1=${NEXUS_DOWNLOAD_OUTPUT_FILE_NAME_SHA1:-"$NEXUS_DOWNLOAD_OUTPUT_FILE_NAME.sha1"}
 echo "NEXUS_DOWNLOAD_OUTPUT_FILE_NAME=$NEXUS_DOWNLOAD_OUTPUT_FILE_NAME"
 echo "NEXUS_DOWNLOAD_OUTPUT_FILE_NAME_SHA1=$NEXUS_DOWNLOAD_OUTPUT_FILE_NAME_SHA1"
-SONATYPE_URL="https://repository.sonatype.org";
+SONATYPE_REPOSITORY=${SONATYPE_REPOSITORY:-public}
```

Comment:
Eredeileg ez "central-proxy" volt de mondjuk nem latok ilyen a URL-en mar, ez a public viszont szinten nem latszik a "https://oss.sonatype.org"-ben

Reply:
mukodott publiccal (azt gondoltam, hogy az tartalmazza a release+snapshotot is)
atirom release-re a defaultot

Reply:
<img width="1213" alt="image" src="https://github.com/i-Cell-Mobilsoft-Open-Source/docker-base/assets/5455213/1578ee99-cec8-4186-8f09-6301cbb380a5">
itt van amugy

Reply:
Valamit beneztem, most a regi "https://repository.sonatype.org/" nem tartalmaz mar semmit csak valami "forge" repot es a "https://oss.sonatype.org" az amiben ott a public. Tehat ha a bongeszobol indulunk ki akkor a "https://oss.sonatype.org" megis a helyes

Reply:
javitottam az adocban is a defaultot